source/fundamentals/crud/read-operations/distinct.txt

```diff
@@ -21,20 +21,23 @@ across a collection.
 Sample Documents
 ~~~~~~~~~~~~~~~~
 
-Consider a collection called ``restaurants`` that contains documents with restaurant data.
-The examples on this page use the following ``restaurants`` documents to demonstrate the
-usage of ``distinct()``:
+Use the following code snippet to insert documents with restaurant
+data to the ``myDB.restaurants`` collection to follow along with the
+examples in this guide:
 
-.. code-block:: json
+.. code-block:: javascript
+
+   const myDB = client.db("myDB")
```

Comment:
I: My JS is rusty, but should this line end with a semicolon?

```suggestion
   const myDB = client.db("myDB");
```

Reply:
changed all